src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -627,6 +684,9 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 	public updateResult(resultSet: ResultSetSummary) {
 		this._resultSet = resultSet;
 		if (this.table && this.visible) {
+			if (this.options.inMemoryDataProcessing && this.options.inMemoryDataCountThreshold < resultSet.rowCount) {
+				this.filterPlugin.enabled = false;
```

Comment:
Is there some kind of indication to the user when this threshold is reached? It would probably be nice to have a popup letting them know that sorting/filtering is disabled and how to increase the limits

Reply:
this is a todo work I am tracking in the main tracking issue.